app/view_models/admin/needs_attention_auctions_view_model.rb

```diff
@@ -12,7 +12,7 @@ def drafts_partial
   end
 
   def drafts
-    @_drafts ||= Auction.unpublished.map do |auction|
+    @_drafts ||= AuctionQuery.new.unpublished.map do |auction|
```

Comment:
not sure we need to call this on `AuctionQuery` -- we have it defined as a scope in `app/models/concerns/auction_scopes.rb` -- is there a good reason to define it on AuctionQuery too?